api/README.md

```diff
@@ -1,32 +1,41 @@
 # The API package
 
-This package sets up a [Express](https://expressjs.com/) API server and a connection to a SQL database using [Knex](https://knexjs.org/).
+This package sets up a [Express](https://expressjs.com/) API server and a connection to a database (SQLite by default) using [Knex](https://knexjs.org/).
 
-For development you can run the command `npm run dev` which uses `nodemon` to watch files and restarts the server when a change happens.  
-You can visit [http://localhost:3001/api](http://localhost:3001/api) which will test the connection to the database.
+For development you can run the command `npm run dev` which uses `nodemon` to watch files and restarts the server when a change happens. You can find the API at [http://localhost:3001/api](http://localhost:3001/api). 
 
-There is no build step so when deploying it is enough to run `npm run start`.
+There is an example route set up at "/" which you can implement to quickly test the connection to the database.
+
+There is no build step, so when deploying it is enough to just run `npm run start`.
 
 ## Environment variables
 
-You can set environment variables in the `.env` file or in the Render.com environment variables section.  
-When you start a fresh project, make sure to copy the `.env` file by using `cp .env-example .env`.
+You can set environment variables in the `.env` file or in the Render.com environment variables section.
+
+When you start a fresh project, check out `.env-template` to get started. Create a file called `.env` and copy the contents of the template as a starting point (or just run `cp .env-template .env`). You should comment in/out the sections you need, and add any additional configuration as necessary.
 
 ## Database clients
 
-The package comes installed with both a MySQL client for local development and a PostgreSQL client to use on [Render.com](https://render.com).  
-You can change which client to use by changing the `DB_CLIENT` environment variable (either `mysql2` or `pg`).
+The package comes installed with an SQLite, MySQL, and PostgreSQL client. Here's a quick suggestion for use cases:
+1. SQLite for quick, simple file-based storage
+2. MySQL for more advanced data storage (requires you to run a database service)
+3. PostgreSQL, similar to MySQL and used on our recommended hosting platform Render.com
+
+You can decide which client to use by changing the `DB_CLIENT` environment variable. See `.env-template` for more info. 
 
-## Database managed with code
+## Advanced database management
 
 You can get far with a simple `.sql` file to manage your database but if you'd prefer to manage your database with Knex, you can use [Knex Migrations](https://knexjs.org/guide/migrations.html) to set up your schema (as well as rollback schema changes across versions).  
+
 You can also use [Knex Seeds](https://knexjs.org/guide/migrations.html#seed-files) to populate your database with data.  
```

Comment:
I totally forgot about database seeds! This could be super useful for DB deployment, although it's another new concept for trainees.